bot/exts/recruitment/talentpool/_review.py

```diff
@@ -55,17 +55,17 @@ async def reschedule_reviews(self) -> None:
         log.trace("Rescheduling reviews")
         await self.bot.wait_until_guild_available()
         # TODO Once the watch channel is removed, this can be done in a smarter way, e.g create a sync function.
-        await self._pool.fetch_user_cache()
+        await self._pool.refresh_cache()
 
-        for user_id, user_data in self._pool.watched_users.items():
+        for user_id, user_data in self._pool.cache.items():
             if not user_data["reviewed"]:
                 self.schedule_review(user_id)
 
     def schedule_review(self, user_id: int) -> None:
         """Schedules a single user for review."""
         log.trace(f"Scheduling review of user with ID {user_id}")
 
-        user_data = self._pool.watched_users.get(user_id)
+        user_data = self._pool.cache[user_id]
```

Comment:
Does this and the lines below reintroduce the bug fixed in https://github.com/python-discord/bot/pull/1569

Reply:
I haven't experienced them myself, but I will change to `.get` just in case, after testing.

Reply:
Resolved in d1a2d9965fadc5292685d3ca78a75fd16f07af26